Avalonia.HtmlRenderer.csproj

```diff
@@ -110,6 +110,6 @@
     <Compile Include="Utilities\Util.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Avalonia" Version="0.10.0" />
+    <PackageReference Include="Avalonia" Version="0.10.13" />
```

Comment:
It is not required but I found it useful also because the NuGet package is broken anyway, so probably no one is using it. But if you find it better we can remove it.

Reply:
0.10.0 is the api...

doesnt matter if you target 0.10.1 or 0.10.2 or 0.10.13... except to restrict the end users.